src/goto-programs/goto_program.cpp

```diff
@@ -32,51 +37,80 @@ std::ostream& goto_programt::output_instruction(
   std::ostream& out,
   instructionst::const_iterator it) const
 {
-  out << "        // " << it->location_number << " ";
+  return output_instruction(ns, identifier, out, *it);
```

Comment:
There are 25 instances of it, but it should be a case of replacing `it` with `*it` so shouldn't take very long, I'll put it as a separate PR when I get a spare 40min.